src/core/qgsfeaturefiltermodel.h

```diff
@@ -107,11 +107,14 @@ class CORE_EXPORT QgsFeatureFilterModel : public QgsFeaturePickerModelBase
      */
     void extraIdentifierValuesChanged();
 
-  private:
-    QgsFeatureExpressionValuesGatherer *createValuesGatherer( const QgsFeatureRequest &request ) const override;
+  protected:
 
     void requestToReloadCurrentFeature( QgsFeatureRequest &request ) override;
 
```

Comment:
Why not re-add the private tag here?

Reply:
I was not aware of the SIP_FORCE macro, I changed my PR